vocabularyCollection/rtqc_method.md

```diff
@@ -0,0 +1,9 @@
+| rtqc_method | definition | status |
+|:---------|:-------|:--------|
+| QUARTOD | https://ioos.noaa.gov/project/qartod/[url] | pending |
+| ego quality control manual | http://doi.org/10.13155/51485[DOI] | pending |
+| SOCIB QC manual | https://doi.org/10.25704/q4zs-tspv[DOI] | pending |
+| gliders tools – SOCCO | https://doi.org/10.3389/fmars.2019.00738[DOI] | pending |
+| UEA Seaglider toolbox | | pending |
```

Comment:
UEA Seaglider toolbox is available here https://bitbucket.org/bastienqueste/uea-seaglider-toolbox/src/toolbox/

Reply:
Thanks Callum.
I'll add this to the branch. 
As Emma said below, we are working on another branch on vocab section together. This will be reflect there.